readme.md

```diff
@@ -494,6 +494,16 @@ vim.cmd("colorscheme nightfox")
 require('lualine').setup({ ... })
 ```
 
+### [Lightline]
+
+Lightline check the value of `vim.g.lightline.colorscheme`. It is not depend on
+the position of `:colorscheme` command, Please set it.
+
+```lua
+vim.cmd("colorscheme nightfox")
+vim.g.lightline = { colorscheme = "nightfox" }
+```
+
```

Comment:
Since setting the colorscheme's value to `nightfox` will only set the bar to `nightfox's` colors some people might think that setting it for nightfox will apply the actual style applied (nordfox, dusk, tera....). This should note that the value should bet set to the name of the style wanted by the user.